services/workflows-service/src/transaction/transaction.controller.external.ts

```
@@ -394,9 +411,60 @@ export class TransactionControllerExternal {
+              select: {
+                correlationId: true,
+                firstName: true,
+                lastName: true,
+              },
+            },
+          },
+        },
       },
     });
+
+    return [];
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)